outputs.tf

```diff
@@ -17,3 +17,13 @@ output "dl_queue_url" {
   description = "Dead letter queue url."
   value       = one(aws_sqs_queue.dlqueue[*].url)
 }
+
+output "lambda_function_arn" {
+  description = "Lambda function arn."
+  value       = aws_lambda_function.this[*].lambda_function_arn
+}
+
+output "lambda_function_name" {
+  description = "Lambda function name."
+  value       = aws_lambda_function.this[*].lambda_function_name
```

Comment:
The user already knows about name since we pass var.name. Not sure if it's worth including an output. If so it will just be the value of var.name.

Reply:
This is true, but I guess it just depends where we want to delegate having to update the name. E.g. right now all of our lambda names are just concatenated strings. We could move those into locals then refer to the local for all of our lambda names, or refer to the output of the lambda. Just want to avoid having 5x copies of the same concatenated string.

Reply:
That's fair. I guess it's just a callout then that this won't work as there's no attribute reference for this.